Makefile

```diff
@@ -299,6 +299,16 @@ quickstart: $(KUSTOMIZE) manifests #EXHELP Generate the installation release man
 
 ##@ Docs
 
+.PHONY: crd-ref-docs
+API_REFERENCE_FILENAME := operator-controller-api-reference.md
+crd-ref-docs: $(CRD_REF_DOCS)
+	rm -f $(ROOT_DIR)/docs/refs/api/$(API_REFERENCE_FILENAME)
+	$(CRD_REF_DOCS) --source-path=$(ROOT_DIR)/api \
+	--config=$(ROOT_DIR)/config/base/crd/bases/olm.operatorframework.io_clusterextensions.yaml \
+	--renderer=markdown \
+	--output-path=$(ROOT_DIR)/docs/refs/api/
```

Comment:
I did some digging in the source code and it looks like they support this being a path to a file as well - could you verify if setting `--output-path=$(ROOT_DIR)/docs/refs/api/$(API_REFERENCE_FILENAME)` would allow us to drop the `mv` operation after this?

Reply:
Ah - you're right. I did try that before but I must have formatted it wrong or had some other issue; I assumed it didn't work. Thanks for digging in to find that! I've updated the PR.